style.css

```diff
@@ -1007,24 +1015,76 @@ button :disabled {
   color: #ff6b00;
 }
 
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+.barr {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
+}
+
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).So Kindly work on styling the mobile menu Just like the Figma design and adjust the background color  of mobile menu just like the Figma design .

- The Figma design Mobile Menu Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/210414877-5eaaaf09-aa4f-46a2-a386-f6e97152a625.PNG)

- Your Project Mobile Menu Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/210415099-363865f6-d218-4bf7-80a9-41a3676f7835.PNG)